xapi.opam

```diff
@@ -28,6 +28,7 @@ depends: [
   "rrdd-plugin"
   "sexpr"
   "sha"
+  "xxhash"
```

Comment:
I think `xxhash` needs to be added to `xs-opam`, or included as a subdir in xapi (probably add to xs-opam + add patch file for the issue we know about would be the way forward).

Reply:
Agreed ("vendoring" in dune can be done by just dropping xxhash into a subdirectory in `ocaml/`)

Reply:
Sounds good, but on the long term I think we should merge back to upstream.